polkadot-launch/launch-rococo-local-with-integritee.json

```diff
@@ -1,6 +1,6 @@
 {
 	"relaychain": {
-		"bin": "../../bin/polkadot-0.9.8",
+		"bin": "../../paritytech/polkadot/target/release/polkadot",
```

Comment:
this appears inconsistent with the other paths

Reply:
my bad: my local settings. I will remove it

Reply:
I'll do